src/fable-standalone/src/Main.fs

```diff
@@ -265,7 +266,8 @@ let init () =
                 if x.StartsWith("--define:") || x.StartsWith("-d:")
                 then x.[(x.IndexOf(':') + 1)..] |> Some
                 else None) |> Array.toList
-            let options = Fable.CompilerOptionsHelper.Make(define=define, ?typedArrays=typedArrays, ?typescript=typescript)
+            let language = typescript |> Option.map (fun ts -> match ts with | true -> TypeScript | _ -> JavaScript) |> Option.defaultValue JavaScript
+            let options = Fable.CompilerOptionsHelper.Make(define=define, ?typedArrays=typedArrays, language=language)
```

Comment:
@dbrattli Is this missing the `?` on the language arg?

Reply:
No, don't think so. The `language` is not an option as I use `defaultValue`. Btw. Tried first to propagate language settings but problems with deps and where the Language union should live to be visible for all projects. Currently it's in `Fable.AST/Plugins.fs` which makes it hard to get it into e.g Worker.fsproj. So decided to keep the `typescript: bool` for now for fable-standalone.

Reply:
@dbrattli That's fine, I'll fix it after this gets merged. Thanks!

Reply:
@ncave Sounds great!